tests/unit/test_dataframe.py

```diff
@@ -1191,6 +1185,7 @@ def test_append_inplace_preexisinting_nan(test_scm_df):
         res.timeseries().reorder_levels(exp.index.names).sort_index().reset_index(),
         exp.sort_index().reset_index(),
         check_like=True,
+        check_dtype=False,
```

Comment:
do they both come back as numeric types? Or has one been coerced to a string or something equally unworkable?

Reply:
The column of nan's is coerced to an `object` dtype instead of float in <1.1. Potentially due to 

```
Bug in DataFrame.reset_index() and Series.reset_index() would not preserve data types on an empty DataFrame or Series with a MultiIndex (GH19602)
```